services/secret-service/server/src/main/java/org/apache/airavata/mft/secret/server/AppConfig.java

```diff
@@ -19,7 +19,7 @@
 
 import org.springframework.context.annotation.Configuration;
 
-@Configuration
+@Configuration("SSAppConfig")
```

Comment:
When there are multiple App Configs coming from different services, either their class names should be different or we have to pass a name using the annotation. Otherwise, there will be conflicts with bean definitions due to the same name(default will be "AppConfig") and the same class name